### ISSUE TITLE
CI: Add new core pathogens to pathogen-build-ci

### ISSUE THREAD

**Issue by joverlee521** [nextstrain/augur#1703]
We test pathogens as part of Augur CI to ensure changes don't break automated pathogen builds. 
We've recently added multiple pathogen repos that are not being tested yet:

- [ ] hmpv
- [ ] oropouche
- [ ] rabies
- [ ] wnv
- [ ] yellow-fever

The hmpv repo will need to be updated to conform to the `pathogen-repo-ci`, but everything else can be directly added.

**Comment by joverlee521** [comment]
Ah, with upcoming https://github.com/nextstrain/docs.nextstrain.org/issues/241, we might also want to visit https://github.com/nextstrain/augur/issues/1350.

**Comment by victorlin** [comment]
Should similar issues be created for [conda-base](https://github.com/nextstrain/conda-base/blob/0d1c067e1970d8ebdcb05ad26feb3b89f7d78caf/.github/workflows/ci.yaml#L176-L181) and [docker-base](https://github.com/nextstrain/docker-base/blob/45129a0009123b772676cd25b3740136169db999/.github/workflows/ci.yml#L253-L258)?

**Comment by joverlee521** [comment]
> Should similar issues be created for [conda-base](https://github.com/nextstrain/conda-base/blob/0d1c067e1970d8ebdcb05ad26feb3b89f7d78caf/.github/workflows/ci.yaml#L176-L181) and [docker-base](https://github.com/nextstrain/docker-base/blob/45129a0009123b772676cd25b3740136169db999/.github/workflows/ci.yml#L253-L258)?

Yes! Good catch, created issues for [conda-base](https://github.com/nextstrain/conda-base/issues/101) and [docker-base](https://github.com/nextstrain/docker-base/issues/238)

**Comment by joverlee521** [comment]
Augur v32.0.0 caused [norovirus phylo builds](https://github.com/nextstrain/norovirus/actions/runs/18724613851) to fail which made me realize norovirus is not included in the augur CI. I updated the list in the main issue with more recently added pathogens.
